docs/rootless.md

```diff
@@ -69,12 +133,24 @@ Remarks:
 
 ### Client
 
-You can just use the upstream Docker client but you need to set the socket path explicitly.
+You need to set the socket path explicitly.
 
 ```console
-$ docker -H unix://$XDG_RUNTIME_DIR/docker.sock run -d nginx
+$ export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/docker.sock
+$ docker run -d nginx
 ```
 
+### Rootless Docker in Docker
+
+To run Rootless Docker inside "rootful" Docker, use `docker:<version>-dind-rootless` image instead of `docker:<version>-dind` image.
+
+```console
+$ docker run -d --name dind-rootless --privileged docker:19.03-dind-rootless --experimental
+```
+
+`docker:<version>-dind-rootless` image runs as a non-root user (UID 1000).
+However, `--privileged` is required for disabling seccomp, AppArmor, and mount masks.
```

Comment:
🤔 we could create a script that directly calls the API, and disables mount masks.

Could actually be a CLI plugin 🤔🤔

Reply:
we already have `--security-opt systempaths=unconfined`, but seems not enough

```
$ docker run --rm --security-opt seccomp=unconfined --security-opt apparmor=unconfined --security-opt systempataths=unconfined --cap-add all docker:19.03.3-dind-rootless --experimental
...
mount: permission denied (are you root?)
time="2019-10-13T11:15:55Z" level=warning msg="failed to mount sysfs ([[mount -t sysfs none /sys]]), falling back to read-only mount ([[mount -t sysfs -o ro none /sys]]): exit status 1"
open: No such file or directory
[rootlesskit:child ] error: executing [[ip tuntap add name tap0 mode tap] [ip link set tap0 address 02:50:00:00:00:01]]: exit status 1
```

Reply:
Interesting 🤔